### PR TITLE
Added UwU channel 

### PR DESCRIPTION
**Pull request by woo200**
When you use /uwu_channel or +uwu_channel it will add a channel, where everything in that channel is uwuified via a webhook

### CODE REVIEW

**Review by diminDDL — CHANGES_REQUESTED** [review]
Move all the uwu stuff in to the uwu cog please.

**Comment by woo200** [comment]
> Move all the uwu stuff in to the uwu cog please.

ironic because I initially had it all in the uwu cog but then i thought it looked weird so I moved it to funstuff :\

**Review by diminDDL — APPROVED** [review]
Seems legit

**Review by ThatRedKite — CHANGES_REQUESTED** [review]
please dont delete the `__init__.py`

File: thatkitebot/cogs/uwucog.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ async def uwuify(message: str):
     return message.replace('na', 'nya').translate(trans_table).replace("no", "yo").replace("mo", "yo").lower()+' uwu'
 
 # Yes this definately needs its own cog shut the fuck up kite (Jk I love you)
-class UwuCog(commands.Cog, name=""):
```
Comment (woo200) [inline thatkitebot/cogs/uwucog.py]: Name was set to "" which caused help command to fail when it tried to send the empty name in a discord embed

**Review by ThatRedKite — APPROVED** [review]
This is a horrible idea but here you go

**Pull request merged**